code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -154,6 +154,8 @@
 	else if(istype(W, /obj/item/stack/packageWrap))
 		return
 	else if(istype(W, /obj/item/stack/cable_coil))
+		if(isrobot(user))

```

Comment:
This is a disgusting hack BTW; you should really find out what the underlying issue is and fix that, rather than applying bandaid solutions like this.

Plus, if other things exhibit similar behaviour like this in the future, you'll just have to apply more bandaids like this instead of fixing the actual code that breaks it

Reply:
Fair point. Looking at it a bit better I know another fix. Bit to zealous here :P